resource_customizations/gateway.networking.k8s.io/GRPCRoute/health.lua

```diff
@@ -0,0 +1,64 @@
+local hs = {}
+
+function checkConditions(conditions, conditionType)
+  for _, condition in ipairs(conditions) do
+    if condition.type == conditionType and condition.status == "False" then
+      return false, condition.message or ("Failed condition: " .. conditionType)
```

Comment:
Should we have a test for the case where condition.message is nil?

Reply:
I appreciate so much for your review! Yes, it should be added for testing nil guard